src/components/SingleProductPage/SingleProduct.js

```diff
@@ -0,0 +1,79 @@
+import "./SingleProduct.css";
+import { useParams, useHistory } from "react-router";
+import useProducts from "../../hooks/useProducts";
+import getProductById from "../../utility/getProductById";
+import { FaRegSnowflake, FaArrowLeft } from "react-icons/fa";
+import { BsFillClockFill } from "react-icons/bs";
+
+export default function SingleProduct() {
+  const { singleProductId } = useParams();
+  const products = useProducts();
+  const history = useHistory();
+
+  if (products.length !== 0) {
+    const productInfo = getProductById(Number(singleProductId), products);
+    const {
+      name,
+      image,
+      ingredients,
+      time,
+      refrigeration,
+      goals,
+    } = productInfo;
+
+    const classForTonerImage =
+      name === "Glycolic Acid 7% Toning Solution" ? "toner-image" : "";
+    return (
+      <div className="SingleProduct">
+        <img
+          src={image}
+          alt={productInfo}
+          className={`single-product-image ${classForTonerImage}`}
+        />
+        <h3 className="single-product-name">{name}</h3>
+        <div className="single-product-info">
+          {refrigeration && (
+            <span className="single-product-refrigeration">
+              <span className="single-product-refrigeration-icon">
+                <FaRegSnowflake />
+              </span>{" "}
+              <span className="single-product-refrigeration-text">
+                Please refrigerate{" "}
```

Comment:
```suggestion
                Please refrigerate
```

Reply:
starting to wonder where these come from 😂

Reply:
I think this is the "auto" correction from vsc. You had some space in it and vsc put this space in `{" "}`
It happens to the best of us ;-)

Reply:
ah well then it makes sense it happened to me 😁